docs/source/index.rst

```diff
@@ -44,6 +44,7 @@ Kale APIs above are ordered following the machine learning pipeline, i.e., funct
     CIFAR - ISONet <https://github.com/pykale/pykale/tree/main/examples/cifar_isonet>
     Digits - Domain Adaptation <https://github.com/pykale/pykale/tree/main/examples/digits_dann>
     Digits/Office - Multisource Adaptation <https://github.com/pykale/pykale/tree/main/examples/office_multisource_adapt>
+    Multimodal - AVMNIST <https://github.com/pykale/pykale/tree/main/examples/avmnist_multimodal>
```

Comment:
Your name is not following the sematics of others [ data - method ]. Please learn from other examples to ensure consistency.

Reply:
test

Reply:
changed from Multimodal - AVMNIST to AVMNIST - Multimodal Deep Learning.